index.js

```diff
@@ -0,0 +1,67 @@
+const { Client, GatewayIntentBits } = require('discord.js');
+const axios = require('axios');
+const client = new Client({ intents: [GatewayIntentBits.Guilds, GatewayIntentBits.GuildMessages, GatewayIntentBits.MessageContent] });
+const fs = require('fs');
+
+//config로 토큰값 관리
+const config = JSON.parse(fs.readFileSync('config.json', 'utf8'));
+
+// 봇 토큰
+const token = config.token;
+
+// 명언 API URL
+const quoteAPI = 'https://api.quotable.io/random';
```

Comment:
API URL과 같은 상수는 변수의 이름을 모두 대문자로 작명을 해주는 것이 가시성을 높여 주는 부분이 있습니다.

```suggestion
const QUOTE_API_URL = 'https://api.quotable.io/random';
```